test/types/cptr/const/c_ptrConst_modify.good

```diff
@@ -0,0 +1 @@
+c_ptrConst_modify.chpl:6: error: cannot assign to const variable
```

Comment:
Do you think it is easy to change this error message? I can help with wording it if it is easy, otherwise, let's move on.

Reply:
I don't think it would be an easy change, because as far as the compiler's concerned this is an attempt to modify a `const ref` just like any other, nothing specific to it being from a `c_ptrConst`.